utils/cfutils.go

```diff
@@ -94,12 +94,18 @@ func checkUserPathAvailability(app string, path string) (bool, error) {
 }
 
 func (checker CfJavaPluginUtilImpl) CheckRequiredTools(app string) (bool, error) {
-	output, err := exec.Command("cf", "ssh-enabled", app).Output()
+	guid, err := exec.Command("cf", "app", app, "--guid").Output()
+	if err != nil {
+		return false, err
+	}
+	output, err := exec.Command("cf", "curl", "/v3/apps/"+strings.TrimSuffix(string(guid), "\n")+"/ssh_enabled").Output()
```

Comment:
Do you already use [`gofmt`](https://pkg.go.dev/cmd/gofmt) in your IDE? Looking at this concatenation without spaces between the `+` it looks like no code formatter was used yet.
Can you please run `gofmt` once on your change?

Reply:
Hi Tim, 
First of all sorry for the late reply, somehow notification email ended up in spam somehow.
I used the VSCode [Go Extension](https://code.visualstudio.com/docs/languages/go) which has a formatter. I have tried the gofmt as well which also eliminates the spaces between concatenation.

BR,
Raheel

Reply:
You're right, my mistake. This is indeed the correct format!